framework/src/modules/nft/cc_commands/cc_transfer.ts

```diff
@@ -70,13 +70,12 @@ export class CrossChainTransferCommand extends BaseCCCommand {
 
 		const nftStore = this.stores.get(NFTStore);
 		const nftExists = await nftStore.has(getMethodContext(), nftID);
-		if (nftChainID.equals(ownChainID) && !nftExists) {
-			throw new Error('Non-existent entry in the NFT substore');
-		}
 
-		const owner = await this._method.getNFTOwner(getMethodContext(), nftID);
-		if (nftChainID.equals(ownChainID) && !owner.equals(sendingChainID)) {
-			throw new Error('NFT has not been properly escrowed');
+		if (nftChainID.equals(ownChainID)) {
+			const owner = await this._method.getNFTOwner(getMethodContext(), nftID);
```

Comment:
Should an update in LIP be requested as well? It suggests `raise Exception("Non-existent entry in the NFT substore")` which is not consistent with other places in LIP that suggests to throw `NFT substore entry does not exist` for the scenario.

Reply:
probably yes